client/src/components/Header.js

```diff
@@ -5,6 +5,8 @@ import { Link } from "react-router-dom"
 import 'semantic-ui-css/semantic.min.css';
 import swal from "sweetalert";
 import { makeCall } from "../apis";
+import FootyPoints from './FootyPoints';
+
```

Comment:
Get outta here with that python bullsh*t

Reply:
Lol I think there's actually some minor performance improvements for using import over require, and I think it's expected to be more of the standard in JS, though it's a newer feature

Reply:
Also fuck you I like python :)

Reply:
Nothing wrong with using `import` that's a welcome ES6 thing, but I mean the 2 line spaces. It's like you don't even try my dude